diff -aurN gnome-power-0.0.5.orig/gnome-power.schemas gnome-power-0.0.5/gnome-power.schemas
--- gnome-power-0.0.5.orig/gnome-power.schemas	2005-06-18 18:33:51.000000000 +0200
+++ gnome-power-0.0.5/gnome-power.schemas	2005-07-05 15:16:41.916326472 +0200
@@ -71,7 +71,7 @@
       <key>/schemas/apps/gnome-power/general/lowThreshold</key>
       <applyto>/apps/gnome-power/general/lowThreshold</applyto>
       <owner>gnome-power</owner>
-      <type>integer</type>
+      <type>int</type>
       <default>20</default>
       <locale name="C">
         <short>The powerdevice low-icon threshold.</short>
diff -aurN gnome-power-0.0.5.orig/gnome-power.schemas.in gnome-power-0.0.5/gnome-power.schemas.in
--- gnome-power-0.0.5.orig/gnome-power.schemas.in	2005-06-18 18:33:51.000000000 +0200
+++ gnome-power-0.0.5/gnome-power.schemas.in	2005-07-05 15:16:33.055673496 +0200
@@ -71,7 +71,7 @@
       <key>/schemas/apps/gnome-power/general/lowThreshold</key>
       <applyto>/apps/gnome-power/general/lowThreshold</applyto>
       <owner>gnome-power</owner>
-      <type>integer</type>
+      <type>int</type>
       <default>20</default>
       <locale name="C">
         <short>The powerdevice low-icon threshold.</short>
