diff -urN gnome-power-manager-2.18.1/data/gnome-power-preferences.desktop.in gnome-power-manager-2.18.1.new/data/gnome-power-preferences.desktop.in
--- gnome-power-manager-2.18.1/data/gnome-power-preferences.desktop.in	2007-03-04 18:53:44.000000000 +0100
+++ gnome-power-manager-2.18.1.new/data/gnome-power-preferences.desktop.in	2007-04-04 22:54:46.000000000 +0200
@@ -6,7 +6,7 @@
 Exec=gnome-power-preferences
 Terminal=false
 Type=Application
-Categories=Settings;X-GNOME-SystemSettings;
+Categories=GTK;GNOME;Settings;X-GNOME-SystemSettings;X-Capplet;
 OnlyShowIn=GNOME;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
diff -urN gnome-power-manager-2.18.1/data/gnome-power-statistics.desktop.in gnome-power-manager-2.18.1.new/data/gnome-power-statistics.desktop.in
--- gnome-power-manager-2.18.1/data/gnome-power-statistics.desktop.in	2007-03-04 18:53:44.000000000 +0100
+++ gnome-power-manager-2.18.1.new/data/gnome-power-statistics.desktop.in	2007-04-04 22:56:54.000000000 +0200
@@ -6,7 +6,7 @@
 Exec=gnome-power-statistics
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;System;X-Red-Hat-Base;
+Categories=GNOME;GTK;System;Monitor;
 OnlyShowIn=GNOME;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
