diff -aurN gnome-power-manager-2.14.1.org/data/gnome-power-preferences.desktop.in gnome-power-manager-2.14.1/data/gnome-power-preferences.desktop.in
--- gnome-power-manager-2.14.1.org/data/gnome-power-preferences.desktop.in	2006-03-23 12:49:05.000000000 +0100
+++ gnome-power-manager-2.14.1/data/gnome-power-preferences.desktop.in	2006-04-09 16:56:31.852544368 +0200
@@ -6,7 +6,7 @@
 Exec=gnome-power-preferences
 Terminal=false
 Type=Application
-Categories=Settings;X-GNOME-SystemSettings;
+Categories=GTK;GNOME;Settings;X-GNOME-SystemSettings;AdvancedSettings;X-Capplet;
 OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-power-manager
