diff -urN gnome-power-manager-2.19.92/applets/brightness/brightness-applet.c gnome-power-manager-2.19.92.new/applets/brightness/brightness-applet.c
--- gnome-power-manager-2.19.92/applets/brightness/brightness-applet.c	2007-08-14 22:29:10.000000000 +0200
+++ gnome-power-manager-2.19.92.new/applets/brightness/brightness-applet.c	2007-09-04 20:56:48.000000000 +0200
@@ -27,6 +27,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <libgnome/gnome-i18n.h>
 #include <panel-applet.h>
 #include <gtk/gtk.h>
 #include <gtk/gtkbox.h>
diff -urN gnome-power-manager-2.19.92/applets/inhibit/inhibit-applet.c gnome-power-manager-2.19.92.new/applets/inhibit/inhibit-applet.c
--- gnome-power-manager-2.19.92/applets/inhibit/inhibit-applet.c	2007-08-14 22:29:10.000000000 +0200
+++ gnome-power-manager-2.19.92.new/applets/inhibit/inhibit-applet.c	2007-09-04 20:57:12.000000000 +0200
@@ -27,6 +27,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <libgnome/gnome-i18n.h>
 #include <panel-applet.h>
 #include <gtk/gtk.h>
 #include <gtk/gtkbox.h>
diff -urN gnome-power-manager-2.19.92/src/gpm-control.c gnome-power-manager-2.19.92.new/src/gpm-control.c
--- gnome-power-manager-2.19.92/src/gpm-control.c	2007-07-27 01:28:18.000000000 +0200
+++ gnome-power-manager-2.19.92.new/src/gpm-control.c	2007-09-04 21:06:43.000000000 +0200
@@ -27,6 +27,7 @@
 #include <stdio.h>
 #include <time.h>
 #include <errno.h>
+#include <libgnome/gnome-i18n.h>
 #include <libgnomeui/gnome-client.h>
 
 #include <string.h>
diff -urN gnome-power-manager-2.19.92/src/gpm-notify.c gnome-power-manager-2.19.92.new/src/gpm-notify.c
--- gnome-power-manager-2.19.92/src/gpm-notify.c	2007-08-03 19:13:36.000000000 +0200
+++ gnome-power-manager-2.19.92.new/src/gpm-notify.c	2007-09-04 20:55:19.000000000 +0200
@@ -34,7 +34,7 @@
 #ifdef HAVE_UNISTD_H
 #include <unistd.h>
 #endif /* HAVE_UNISTD_H */
-
+#include <libgnome/gnome-i18n.h>
 #include <glib/gi18n.h>
 #include <dbus/dbus-glib.h>
 #include <gtk/gtk.h>
